{"author":{"id":"5351a6a8fb78ee61e8b2a7be5386271241d800a3825ae0b2052d0b5b2642d81f"},"ops":[{"type":6,"timestamp":1588497536,"metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdElzc3VlRWRpdDozNzczODIyNDY="},"nonce":"OsfHr05o0Max7djgqwdiJEel+9I=","target":"55ffd01d0e8874a5a44993c4e4b5d4e701f0a4c39c060aefbfe123ebec4cd632","message":"git-bug version: 0.7\ngitlab server: GitLab Community Edition 12.10.2 \n\nI get a bug (or feature) with the `bridge pull` command. `git-bug` do not crash, but stop importing after an error.\n\n```bash\nimport error at id 975: label event creation: removed label: empty\nimported 643 issues and 13 identities... \n```\n\nThen...\n- all issues after the error are missing.\n- Many issue closed are open in git-bug.\n```\nGitLab: Open 76    Closed 697    All 773\ngit-bug: open 640    closed 3\n```\n\nIf I run again the command, issues are added again so I have them twice !\n\n```bash \nimport error at id 975: label event creation: removed label: empty\nimported 643 issues and 0 identities with git.duniter.org bridge\n```\nThen:\n```\nGitLab: Open 76    Closed 697    All 773\ngit-bug: open 1280     closed 6 \n```\n\nWas so close to heaven ! ;)","files":null}]}